src/main/java/com/stuypulse/robot/commands/FunnelFunnelWithMotorStallingCommand.java

```diff
@@ -0,0 +1,44 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package com.stuypulse.robot.commands;
+
+import com.stuypulse.robot.MotorStall;
+import com.stuypulse.robot.subsystems.Funnel;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class FunnelFunnelWithMotorStallingCommand extends CommandBase {
+    
+    private final Funnel m_Funnel;
+
+    public FunnelFunnelWithMotorStallingCommand(Funnel funnel) {
+        m_Funnel = funnel;
+        addRequirements(m_Funnel);
+    }
+
+    @Override
+    public void initialize() {
+        new Thread(new MotorStall(m_Funnel)).start();
```

Comment:
Shouldn't this thread be initialized in robotInit, and then called by all of the potential commands/methods that need its data